ui/app/AppLayouts/Chat/ContactsColumn.qml

```diff
@@ -10,23 +10,233 @@ import "./components"
+    }
+
+    ScrollView {
+        id: chatGroupsContainer
+
+        anchors.top: contactRequests.bottom
         anchors.topMargin: Style.current.padding
+        anchors.bottom: parent.bottom
         anchors.horizontalCenter: parent.horizontalCenter
-        font.weight: Font.Bold
-        font.pixelSize: 17
+
+        width: parent.width
+
+        leftPadding: Style.current.halfPadding
+        rightPadding: Style.current.halfPadding
+
+        ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
+
+        contentHeight: channelList.height + 2 * Style.current.padding + emptyViewAndSuggestions.height + emptyViewAndSuggestions.anchors.topMargin
+        clip: true
+
+        Item {
+            id: noSearchResults
+            anchors.top: parent.top
+            anchors.horizontalCenter: parent.horizontalCenter
+            width: parent.width
+            visible: !!!channelList.height && contactsColumn.searchStr !== ""
+            height: visible ? 300 : 0
+
+            StatusBaseText {
+                font.pixelSize: 15
+                color: Theme.palette.directColor5
+                anchors.centerIn: parent
+                text: qsTr("No search results")
+            }
+        }
+
+        StatusChatList {
+            id: channelList
+
+            chatNameFn: function (chatItem) {
+                return chatItem.chatType !== Constants.chatTypePublic ?
+                    Emoji.parse(Utils.removeStatusEns(Utils.filterXSS(chatItem.name))) :
+                    Utils.filterXSS(chatItem.name)
+            }
+
+            profileImageFn: function (id) {
+                return appMain.getProfileImage(id)
+            }
+
+            filterFn: function (chatListItem) {
+                return !!!contactsColumn.searchStr || chatListItem.name.toLowerCase().includes(contactsColumn.searchStr.toLowerCase())
+            }
+
+            Connections {
+                target: profileModel.contacts.list
+                onContactChanged: {
+                    for (var i = 0; i < channelList.chatListItems.count; i++) {
+                        let chatItem = channelList.chatListItems.itemAt(i);
+                        if (chatItem.chatId === pubkey) {
+                            let profileImage = appMain.getProfileImage(pubkey)
+                            if (!!profileImage) {
+                                chatItem.image.isIdenticon = false
+                                chatItem.image.source = profileImage
+                            }
+                            break;
+                        }
+                    }
+                }
+            }
+
+            chatListItems.model: chatsModel.channelView.chats
+            selectedChatId: chatsModel.channelView.activeChannel.id
+
+            onChatItemSelected: chatsModel.channelView.setActiveChannel(id)
+            onChatItemUnmuted: chatsModel.channelView.unmuteChatItem(id)
+
+            popupMenu: StatusPopupMenu {
```

Comment:
The equivalent of this popup lives currently in `ChannelContextMenu`, which I haven't removed in this PR yet, because it's still used in the `CommunityColumn`.

There's a even chance we'll move this `StatusPopupMenu` into `ChannelContextMenu` for reusability if needed.